modules/rosa-required-aws-service-quotas.adoc

```diff
@@ -8,8 +8,16 @@
 
 The table below describes the AWS service quotas and levels required to create and run one {product-title} cluster. Although most default values are suitable for most workloads, you might need to request additional quota for the following cases:
 
-* ROSA (classic architecture) clusters require a minimum AWS EC2 service quota of 100 vCPUs to provide for cluster creation, availability, and upgrades. The default maximum value for vCPUs assigned to Running On-Demand Standard Amazon EC2 instances is `5`. Therefore if you have not created a ROSA cluster using the same AWS account previously, you must request additional EC2 quota for `Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances`.
-
+* ROSA clusters require a minimum AWS EC2 service quota of
+ifndef::openshift-rosa-hcp[]
+100{nbsp}vCPUs
+endif::[]
+ifdef::openshift-rosa-hcp[]
+32{nbsp}vCPUs
+endif::[]
+to provide for cluster creation, availability, and upgrades. The default maximum value for vCPUs assigned to Running On-Demand Standard Amazon EC2 instances is `5`. Therefore if you have not created a ROSA cluster using the same AWS account previously, you must request additional EC2 quota for `Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances`.
```

Comment:
Is `Running On-Demand Standard (A, C, D, H, I, M, R, T, Z) instances` supposed to be in back ticks? -- if yes, then please ignore my question

Reply:
Unfortunately, yes - this is the exact value for the name of the quota type that they need to add quota for.